Automatic Validation ended with: 
> 2026-06-08T21:52:57.9536721Z ##[error] Manifest Error: Duplicate installer entry found.

(Deterministic automation - build 1385.)

--- a/manifests/h/hasherezade/PE-bear/0.7.2/hasherezade.PE-bear.installer.yaml
+++ b/manifests/h/hasherezade/PE-bear/0.7.2/hasherezade.PE-bear.installer.yaml
@@ -16,9 +16,9 @@ Installers:
   Dependencies:
     PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x64
-- Architecture: x64
-  InstallerUrl: https://github.com/hasherezade/pe-bear/releases/download/v0.7.2/PE-bear_0.7.2_qt5_x64_win_vs17.zip
-  InstallerSha256: A9420CD7F417813E32C762DE83806AF631A48C1E70725FB2DF9A664B3F34977D
+- Architecture: x86
+  InstallerUrl: https://github.com/hasherezade/pe-bear/releases/download/v0.7.2/PE-bear_0.7.2_qt5_x86_win_vs17.zip
+  InstallerSha256: 58431C57CBFFE63348F4E9FE0BE82DA32E0C8B298685A78642307A012CA728DD
   Dependencies:
     PackageDependencies:
     - PackageIdentifier: Microsoft.VCRedist.2015+.x86

